index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Andres -- Martin</title>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="main" class="main">
+      <h1 id="title">Nombre Blog</h1>
+      <p>mensaje titulo</p>
+      <figure id="img-div">
+          <div>
+              <img id="image" src="img/computador.jpg"  />
```

Comment:
I think it would be better to use an alt attribute to the <img> element to describe the image for users who use screen readers.

Reply:
<b>Changes required</b>
I think it would be better to include an alt attribute in the img element to provide a text alternative for users who are visually impaired or who use screen readers. The alt attribute describes the contents of the image and provides context for users who may not be able to see the image. Without an alt attribute, users who rely on screen readers will not be able to access the information conveyed by the image. For example, if the image shows a computer screen, a good alt attribute might be "Computer screen displaying website homepage." This provides valuable information for users who may not be able to see the image, and it also helps with accessibility compliance.